code/modules/cargo/order.dm

```diff
@@ -1,3 +1,9 @@
+#define MANIFEST_ERROR_CHANCE 5
+// cargo error bitflags
+#define MANIFEST_ERROR_NAME (1 << 1)
+#define MANIFEST_ERROR_CONTENTS (1 << 2)
+#define MANIFEST_ERROR_ITEM (1 << 3)
```

Comment:
Also I cleaned up this to show that it was using bitflags since 

```
#define MANIFEST_ERROR_NAME 1
#define MANIFEST_ERROR_CONTENTS 2
#define MANIFEST_ERROR_ITEM 4
```

are hardcoded bitflag values.

Reply:
what you've got here is 2 4 and 8, not 1 2 and 4

Reply:
Thanks for pointing this out.  I forget 2^0 = 1

Reply:
Fixed with https://github.com/timothymtorres/tgstation/commit/9a80b9dcb7261772e1173a833bea18a1375301a0